docs/navigation-advanced.md

```diff
@@ -423,8 +423,9 @@ appSwitcher = {
 Tab-style navigation in Luigi can be displayed directly above the micro frontend area, providing you with additional menu options. When you put tab navigation nodes into a [category](navigation-configuration.md#category), they will be rendered in a drop-down. Add this parameter to your configuration to create tab navigation nodes:
 
 ### tabNav
-- **type**: boolean
+- **type**: boolean or Object
 - **description**: renders the children of the node as a horizontal navigation bar. Sub-children are not supported. When you categorize nodes you will get a drop-down menu in the horizontal navigation.
+When you have only one child in the horizontal navigation bar you can specify an object instead of boolean with `{hideTabNavAutomatically:true}` and the horizontal navigation bar will be hidden automatically.
```

Comment:
It seems to suggest that when:
`tabNav: {hideTabNavAutomatically:true}`

it indirectly means that **tabNav** is automatically assumed  to be set **true**. 

I guess we should somehow state in docu that if you have the following:
`tabNav: {hideTabNavAutomatically:true}`

then it means that **tabNav** is inherently also activated when hideTabNavAutomatically = true

Otherwise it makes it ambiguous, like what happens when `tabNav: { hideTabNavAutomatically: false }`. Although I don't know how we could phrase it in words. Maybe @alexandra-simeonova can help with that.